pkg/CoreFx.Private.TestUtilities/CoreFx.Private.TestUtilities.pkgproj

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" 
+    xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+    <PropertyGroup></PropertyGroup>
```

Comment:
nit: Empty PropertyGroup, also indention is usually 2 spaces in our xml/project files.